modules/cluster-logging-visualizer-indices.adoc

```diff
@@ -33,11 +33,11 @@ The audit logs are not stored in the internal {product-title} Elasticsearch inst
 
 .Procedure
 
-To define index patterns and create visualizations:
+To define index patterns and create visualizations in Kibana:
 
-. Launch Kibana by clicking the Application Launcher {launch} and select *Logging*.
+. In the {product-title} console, click the Application Launcher {launch} and select *Logging*.
 
-. Create your Kibana index patterns:
+. Create your Kibana index patterns by clicking *Management* -> *Index Patterns* -> *Create index pattern*:
 
 ** Regular users must manually create index patterns to see logs for their projects. Users should create a new index pattern named `app` and use the `@timestamp` time field to view their container logs. 
```

Comment:
Regular users -> Users.   All users should create  Index Pattern themselves in 4.5.